### ISSUE TITLE
Request: Mark message as read once copied to clipboard 

### ISSUE THREAD

**Issue by jelias** [SoFriendly/2fhey#4]
I wanted to request a feature if possible!

It would be awesome if once the code is copied to the clipboard it could mark the message as read, so it clears from Notification Center and such. 

Perhaps the logic is, once copied, wait 1min, then mark as read? Perhaps it is a setting people could enable/disable?

**Comment by itsthisjustin** [comment]
Yeah looked into this and we were a bit afraid it would cause the iMessage database to corrupt. I'm gonna look into how Bluebubbles and Airmessage handle it to see if I can find a clean way.

**Comment by theprinceofspace** [comment]
Just adding a comment because I would also love if it market these messages as read.

**Comment by homglol** [comment]
Kindly requesting this feature... Mark as read would be awesome

**Comment by jonmacdonald** [comment]
+1

**Comment by justinesgar** [comment]
+1, would love this.  Apple does have a way that if you are getting a text code into Safari it will actually delete the message from iMessage once the code is used. Maybe something a long those lines?

**Comment by alex-starfish** [comment]
This would be awesome!
